### PR TITLE
Opencast 3182 lti tool path change

### PR DESCRIPTION
**Pull request by npmali**
### Your pull request should…

* [ ] have a concise title
* [ ] [close an accompanying issue](https://help.github.com/en/articles/closing-issues-using-keywords) if one exists
* [ ] [be against the correct branch](https://docs.opencast.org/develop/developer/development-process#acceptance-criteria-for-patches-in-different-versions)
* [ ] include migration scripts and documentation, if appropriate
* [ ] pass automated tests
* [ ] have a clean commit history
* [ ] [have proper commit messages (title and body) for all commits](https://medium.com/@steveamaza/e028865e5791)

### CODE REVIEW

File: modules/lti/src/components/Series.tsx [hunk 1]
```diff
@@ -159,7 +159,7 @@ class TranslatedSeries extends React.Component<SeriesProps, SeriesState> {
         if (typeof qs.series_name === "string")
             seriesSuffix = "&series_name=" + qs.series_name;
         const debugSuffix = typeof qs.debug === "string" ? "&debug=" + qs.debug : "";
-        window.location.href = "/ltitools/index.html?subtool=upload&episode_id=" + id + seriesSuffix + debugSuffix;
+        window.location.href = "/ltitools/uct/index.html?subtool=upload&episode_id=" + id + seriesSuffix + debugSuffix;
```
Comment (smarquard) [inline modules/lti/src/components/Series.tsx]: Not sure we need to change this?

File: etc/security/mh_default_org.xml [hunk 2]
```diff
@@ -344,7 +344,7 @@
     <sec:intercept-url pattern="/lti/**" access="ROLE_OAUTH_USER" />
 
     <!-- Enable access to the LTI tools -->
-    <sec:intercept-url pattern="/ltitools/**" access="ROLE_OAUTH_USER" />
+    <sec:intercept-url pattern="/ltitools/uct/**" access="ROLE_OAUTH_USER" />
```
Comment (smarquard) [inline etc/security/mh_default_org.xml]: This doesn't need to change

**Comment by smarquard** [comment]
Not sure what you're aiming to do here? Is this supposed to move all the UCT-related code to /ltitools/uct/... ? It seems a mixture of changes.

**Review by TurRil — APPROVED** [review]

File: etc/security/mh_default_org.xml (hunk 2 above)
Comment (npmali) [reply etc/security/mh_default_org.xml]: Reverted!

**Pull request merged**